packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -162,6 +164,9 @@ function Breadcrumbs<T>(props: SpectrumBreadcrumbsProps<T>, ref: DOMRef) {
   let breadcrumbItems = childArray.map((child, index) => {
     let isCurrent = index === lastIndex;
     let key = child.key ?? index;
+    // If last breadcrumb item children type is not undefined, that means it isn't a string and is a Heading component
+    // and thus should be multiline
+    let multiline = isCurrent && child.props.children?.type;
```

Comment:
Don't really like how I did this here, but if we are going to assume that Heading is the first/only child to an Item, then this might be ok. Open to suggestions though. Perhaps I could have a useEffect that does the same `lastItem.firstChild instanceof HTMLHeadingElement` as above and sets a ref.current to turn on the multiline style?